File: beecrowd/1/1037/1037.clj
(let [num (read)]
  (cond
    (and (>= num 0) (<= num 25))  (println "Intervalo [0,25]")
    (and (> num 25) (<= num 50))  (println "Intervalo (25,50]")
    (and (> num 50) (<= num 75))  (println "Intervalo (50,75]")
    (and (> num 75) (<= num 100)) (println "Intervalo (75,100]")
    :else (println "Fora de intervalo")))

File: beecrowd/1/1037/1037.c
#include <stdio.h>

int main(void) {
    double num;
    scanf("%lf", &num);
    if (num >= 0 && num <= 25) {
        puts("Intervalo [0,25]");
    } else if (num > 25 && num <= 50) {
        puts("Intervalo (25,50]");
    } else if (num > 50 && num <= 75) {
        puts("Intervalo (50,75]");
    } else if (num > 75 && num <= 100) {
        puts("Intervalo (75,100]");
    } else {
        puts("Fora de intervalo");
    }
    return 0;
}
